doc/userguide/install.rst

```diff
@@ -267,10 +267,10 @@ To use backports, the backports repository for the current stable
 distribution needs to be added to the system-wide sources list.
 For Debian 10 (buster), for instance, run the following as ``root``::
 
-    echo "deb http://http.debian.net/debian buster-backports main" > \
+    sudo echo "deb http://http.debian.net/debian buster-backports main" > \
         /etc/apt/sources.list.d/backports.list
```

Comment:
This particular command will actually fail when sudo is added, instead it should probably be:
```
echo "deb http://http.debian.net/debian buster-backports main" | sudo tee -a /etc/apt/sources.list.d/backports.list
```

Reply:
You're right! Thank you for the feedback.